cloud_cleaner/config.py

```diff
@@ -38,13 +37,28 @@ def __init__(self, parser=None, args=None):
                 "(up to 2 times)"
         self.__parser.add_argument("-v", "--verbose", help=_help,
                                    action='count', default=0)
+        _help = "Email warning messages to server creators if their server. " \
+                "is to be deleted. By default, the command does not email. " \
+                "Add this switch to include email."
+        self.__parser.add_argument("--email", "-e",
+                                   help=_help,
+                                   action='store_true')
+        _help = "The email address that should be used to send emails. Only " \
+                "used if --email is set. Required if --email is set."
+        self.__parser.add_argument("--sender", help=_help, default="")
+        _help = "The smtp server name which should be used to send emails. " \
+                "Only used if --email is set. Required if --email is set"
+        self.__parser.add_argument("--smtpN", help=_help, default="")
+        _help = "The smtp server port which should be used to send emails. " \
+                "Only used if --email is set. Required if --email is set"
+        self.__parser.add_argument("--smtpP", help=_help, default=0)
```

Comment:
I wonder if it would look cleaner if we pushed these strings out to somewhere else. Something like:

```python
help_strings = {
    "email": "...",
    "sender": "...",
    ...
}
```

Then we could just do help=help_strings["email"] here. The "help_strings" variable could even be in a separate file that gets imported at the top `from help_strings import help_strings`?

Not a requirement, just an idea.